spec/features/work_order_cancellation_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+
+RSpec.describe 'Work Order Cancellation' do
+  include Helpers::Authentication
+
+  scenario 'Cancel a Work Order correctly', :js do
+    sign_in
+
+    # FIXME: s/hackneyrepairs/#{ ENV["..."] }/g
+
+    #
+    # stub minimum possible
+    #
+    stub_request(:get, "https://hackneyrepairs/v1/work_orders/00000666")
```

Comment:
Might be nice to say what these are doing so if we come back to it we know what's going on rather than working it out all over again?

Reply:
what do you mean? like saying `# get work order 00000666` and so on?

Reply:
yeah. I guess not super important but if stuff changes it's going to be a nightmare going back through those

Reply:
done! do you think it's enough?

Reply:
Looks good to me!